src/strategies/silo/SiloStrategy.sol

```diff
@@ -157,15 +186,17 @@ contract SiloStrategy is BaseHealthCheck, TradeFactorySwapper {
         override
         returns (uint256 _totalAssets)
     {
+        silo.accrueInterest(address(asset));
+
         // Only harvest and redeploy if the strategy is not shutdown.
         if (!TokenizedStrategy.isShutdown()) {
-
-            // Check how much we can re-deploy into the yield source.
-            uint256 toDeploy = asset.balanceOf(address(this));
-            // If greater than 0.
-            if (toDeploy > 0) {
-                // Deposit the sold amount back into the yield source.
-                _deployFunds(toDeploy);
+            uint256 _toDeploy = asset.balanceOf(address(this));
+            if (_toDeploy > 0) {
+                if (_toDeploy <= availableDepositLimit(address(0))) {
+                    _deployFunds(_toDeploy);
```

Comment:
If `toDeploy` is lesser than max deposit then deposit the max deposit instead of not depositing anything

Reply:
Added in https://github.com/johnnyonline/yVaults-v3-strategies/pull/8/commits/405be2165fe830fac0ee6b35db81fdb9c347e596.